modules/ocf/manifests/firewall/nfs.pp

```diff
@@ -0,0 +1,69 @@
+class ocf::firewall::nfs {
+  ensure_resources('service', {
+    'nfs-config' => {},
+    'rpcbind'    => {},
+    'quotarpc'   => {},
+    'nfs-server' => {},
+  })
+
+  augeas {
+    default:
+      require => Package['nfs-kernel-server'];
+
+    'Set rpc.statd ports':
+      lens    => 'Shellvars.lns',
+      incl    => '/etc/default/nfs-common',
+      changes => "set STATDOPTS '\"--port 32765 --outgoing-port 32766\"'";
+
+    'Set rpc.rquotad port':
+      lens    => 'Shellvars.lns',
+      incl    => '/etc/default/quota',
+      changes => "set RPCRQUOTADOPTS '\"-p 32769\"'",
+      notify  => Service['quotarpc'];
+
+    'Set nfs-kernel-server port':
+      lens    => 'Shellvars.lns',
+      incl    => '/etc/default/nfs-kernel-server',
+      changes => "set RPCMOUNTDOPTS '\"-p 32767\"'",
+      notify  => Service['nfs-server'];
```

Comment:
These also need to be enabled for `ocf_www`, since death runs a NFS server to share apache logs, and also needs the quota port to be open, otherwise `quota` hangs until it finally times out.

Reply:
I believe this has been done in `ocf_www/manifests/logging.pp`?

Reply:
It appears so, but something is still definitely blocking quota lookups from `www` in the firewall, so maybe something on @kkuehler's puppet environment?

Reply:
To my knowledge, my environment is only running on mirrors and is up to date with master + some systemd timer stuff.

Reply:
Ok, so I went on `death` and found that the `statd` service wasn't listening on the correct ports, so I restarted it and now running `quota` does not hang, although it still gets a connection refused only on tsunami (not on supernova). I couldn't see an instance of `rquotad` running, so I'm not sure why that is, but I think that is pretty normal, there shouldn't be quotas on the apache logs on death anyway.